bestie/server/testresult.go

```diff
@@ -142,77 +141,89 @@ func handleTestResultEvent(bazelBuildEvent bes.BuildEvent, streamId *build.Strea
 		// Attempt to read the zip file failed.
 		return fmt.Errorf("Error reading %s file: %w", outFileName, readErr)
 	}
```

Comment:
It's a bit easier to verify correctness if the close happens in the same function, close to where the open happened (via `defer()`)

In this case - maybe `defer zipFileCloser.Close()` right after the switch statement, and then `extractZipFiles` can take an `io.Reader`, and doesn't have to close

Reply:
OK, makes sense.